easybuild/easyblocks/generic/cmdcp.py

```diff
@@ -68,7 +68,12 @@ def build_step(self):
             # find (first) regex match, then complete matching command template
             cmd = None
             for regex, regex_cmd in self.cfg['cmds_map']:
-                if re.match(regex, os.path.basename(src)):
+                try:
+                    regex = re.compile(regex)
```

Comment:
actually, wouldn't it be better to use a different variable name for the compiled regex?